### PR TITLE
GOBBLIN-1417: Make row batch size in ORC Mapreduce writer configurable

### PR DESCRIPTION
**Pull request by sv2000**
Dear Gobblin maintainers,

Please accept this PR. I understand that it will not be reviewed until I have checked off all the steps below!

### JIRA
- [x] My PR addresses the following [Gobblin JIRA](https://issues.apache.org/jira/browse/GOBBLIN/) issues and references them in the PR title. For example, "[GOBBLIN-XXX] My Gobblin PR"
    - https://issues.apache.org/jira/browse/GOBBLIN-1417

### Description
- [x] Here are some details about my PR, including screenshots (if applicable):
Current behavior enforces the default row batch size of 1024. To prevent OOMs during writes, this config may need to be overridden.

### Tests
- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
Tested with a MR job and ensuring the config is overridden.

### Commits
- [x] My commits all reference JIRA issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
    1. Subject is separated from body by a blank line
    2. Subject is limited to 50 characters
    3. Subject does not end with a period
    4. Subject uses the imperative mood ("add", not "adding")
    5. Body wraps at 72 characters
    6. Body explains "what" and "why", not "how"

### CODE REVIEW

**Comment by sv2000** [comment]
@autumnust Please review.

File: gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/orc/OrcKeyCompactorOutputFormat.java [hunk 1]
```diff
@@ -32,14 +31,19 @@
 import org.apache.orc.mapreduce.OrcMapreduceRecordWriter;
 import org.apache.orc.mapreduce.OrcOutputFormat;
 
-import static org.apache.gobblin.compaction.mapreduce.CompactorOutputCommitter.*;
+import lombok.extern.slf4j.Slf4j;
+
+import org.apache.gobblin.compaction.mapreduce.CompactorOutputCommitter;
+import org.apache.gobblin.writer.GobblinOrcWriter;
+
+import static org.apache.gobblin.compaction.mapreduce.CompactorOutputCommitter.COMPACTION_OUTPUT_EXTENSION;
```
Comment (autumnust) [inline gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/orc/OrcKeyCompactorOutputFormat.java:39]: This doesn't seem to be needed.

**Review by autumnust — CHANGES_REQUESTED** [review]
minor comments

File: gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/orc/OrcKeyCompactorOutputFormat.java (hunk 1 above)
Comment (sv2000) [reply gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/orc/OrcKeyCompactorOutputFormat.java:39]: It is needed in the getRecordWriter() method. Previous import was: 
import static org.apache.gobblin.compaction.mapreduce.CompactorOutputCommitter.*; which I changed to
import static org.apache.gobblin.compaction.mapreduce.CompactorOutputCommitter.COMPACTION_OUTPUT_EXTENSION;

Let me know, if you want to preserve the original import statement.

**Review by autumnust — APPROVED** [review]
+1

File: gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/orc/OrcKeyCompactorOutputFormat.java (hunk 1 above)
Comment (autumnust) [reply gobblin-compaction/src/main/java/org/apache/gobblin/compaction/mapreduce/orc/OrcKeyCompactorOutputFormat.java:39]: no, explicit import is obviously better here. thanks for fixing this.

**Pull request closed without merging**